services/adminService.js

```diff
@@ -23,36 +25,66 @@ exports.acceptOwnersEmail = async (email, role) => {
   return user
 }
 
-exports.deleteOwner = async (email, role) => {
+exports.deleteOwner = async (id, role) => {
   const user = await User.findOneAndDelete({
-    email: email,
+    _id: id,
     role: role,
   })
 
   if (!user) {
-    throw new ApiError(404, 'Wrong email or user is not a hotel owner')
+    throw new ApiError(404, 'Hotel owner with provided id not found')
   }
 
   return user
 }
 
-exports.deleteUser = async (email, role) => {
+exports.deleteUser = async (id, role) => {
   const user = await User.findOneAndDelete({
-    email: email,
+    _id: id,
     role: role,
```

Comment:
we are deleting owner so we dont need additional argument - role
we can just pass ROLE_USER or whatever it is named in app